app/services/analytics_events.rb

```diff
@@ -1118,6 +1118,12 @@ def idv_gpo_confirm_start_over_visited(**extra)
     track_event('IdV: gpo confirm start over visited', **extra) # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
   end
 
+  # A GPO expiration email was sent to the user
+  # @param [String] user_id UUID of user we sent an email to
+  def idv_gpo_expiration_email_sent(user_id:, **extra)
+    track_event(:idv_gpo_expiration_email_sent, user_id: user_id, **extra)
```

Comment:
Would it be useful to track the Profile id or GpoConfirmationCode id that triggered the email?

Reply:
I debated adding the `code_sent_at` timestamp from confirmation code here. I think that would be potentially useful in Cloudwatch and you could trace it back to a database record if you needed to.

Reply:
Added `code_sent_at` in 54eb141d737c0d4aa888fe510218d1261cd04311